junit-platform-commons/src/main/java/org/junit/platform/commons/util/StringUtils.java

```diff
@@ -70,17 +70,8 @@ public static boolean isNotBlank(String str) {
 	 * @see Character#isWhitespace(int)
 	 */
 	public static boolean containsWhitespace(String str) {
-		if (str == null) {
-			return false;
-		}
+		return str != null && str.codePoints().anyMatch(Character::isWhitespace);
 
```

Comment:
Oh, cool, I'd accidentally missed that blank line! Thanks for pointing it out to me. :)